src/main/kotlin/com/simplymerlin/minigameserver/Server.kt

```diff
@@ -61,6 +65,7 @@ class Server {
     }
 
     private fun initialiseEvents() {
+        globalEventHandler.addChild(PvpExtension.events())
```

Comment:
Don't we want this only to be registered at the beginning of a game?

Reply:
this is your code

Reply:
this is your pr